app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,51 @@
+require 'date'
+# Public
+# Controls the attendance of employees
+# It does the following tasks:
+#   - Mark attendance of a employee/ Employee
+#   - Let admin see who is present today
+class AttendancesController < ApplicationController
+  def index
+    @company = Company.find(session[:current_company_id])
+    @attendances_list = @company.attendances.where(status: 1).order(login_time: :desc)
+  end
+
+  def create
```

Comment:
in create and update function, code repeated its better to make function or callback in model file.

Reply:
this will be changed when merged with others